dist/.magento.env.yaml

```diff
@@ -427,6 +427,17 @@
 #                - "index.php"                                                                                        #
 #                - "index.php/customer/account/create"                                                                #
 #######################################################################################################################
+# TTFB_TESTED_PAGES - list pages you wish to perform time to first byte (TTFB) testing on                             #
+# Magento Version: Magento 2.1.4 and later                                                                            #
+# Default value: - [] (an empty array)                                                                                #
+# Stages: post-deploy                                                                                                 #
+# Example:                                                                                                            #
+#          stage:                                                                                                     #
+#            post-deploy:                                                                                             #
+#              TTFB_TESTED_PAGES:                                                                                     #
+#                - "index.php"                                                                                        #
+#                - "index.php/customer/account/create"                                                                #
```

Comment:
It'll be good to add some page examples with a host name.  Or add this to the description.